packages/server/test/unit/project_spec.js

```diff
@@ -698,6 +698,35 @@ This option will not have an effect in Some-other-name. Tests that rely on web s
       })
     })
 
+    it('escapses %, &', function () {
+      const todosSpec = path.join(this.todosPath, 'tests/sub/a&b%c.js')
+
+      return this.project.getSpecUrl(todosSpec)
+      .then((str) => {
+        expect(str).to.eq('http://localhost:8888/__/#/tests/integration/sub/a%26b%25c.js')
+      })
+    })
+
+    if (process.platform !== 'win32') {
+      // ? is tested here because it's valid in Unix/Linux, but invalid in Windows
+      // https://gist.github.com/doctaphred/d01d05291546186941e1b7ddc02034d3
+      it('escapes ?', function () {
+        const filePath = path.join(__dirname, '../support/fixtures/projects/todos/tests/sub/a?.spec.js')
+
+        fs.writeFileSync(filePath, '')
```

Comment:
I realize now that since this is a unit test, the file doesn't need to exist on the file system. That means we can clean this up, ~delete the `sub/a&b%c.js` fixture~, and make this into a simpler test that doesn't need to check `process.platform`.

Reply:
I guess we still want the `sub/a&b%c.js` fixture for the integration test, so nevermind about that.

Reply:
Done.